tests/validators/tests.py

```diff
@@ -258,6 +258,9 @@
     # Trailing junk does not take forever to reject
     (URLValidator(), 'http://www.asdasdasdasdsadfm.com.br ', ValidationError),
     (URLValidator(), 'http://www.asdasdasdasdsadfm.com.br z', ValidationError),
+    # Very long port does not take forever to verify
+    (URLValidator(), 'https://code.djangoproject.com:' + '9' * (2**24), ValidationError),
+    (URLValidator(), 'https://code.djangoproject.com:' + '0' * (2**24) + '1', None),
```

Comment:
I don't see much value in these tests unless we can estimate and check the actual time. It's not "forever" like in the above examples, it will take few minutes instead of less than 1-2 seconds. There is no chance anyone would notice it in the test results.

Reply:
I did some simple search and didn't find a proper way to add timeout to unittest without introducing extra dependency or add bunch of new code. Guess I'll remove these test.